src/Livewire/Servers.php

```diff
@@ -14,6 +17,10 @@
 #[Lazy]
 class Servers extends Card
 {
+    use InteractsWithTime;
+
+    public int|DateTime|null $ignoreAfter = null;
```

Comment:
Livewire does not support `DateInterval` as a prop value, so I've excluded it from the type here.